package.json

```diff
@@ -20,7 +20,8 @@
     "prettier:write": "prettier --write 'packages/**/*.{js,json}'",
     "publish-release": "yarn lerna publish --conventional-graduate",
     "publish-prerelease": "yarn lerna publish prerelease --preid beta --dist-tag next",
-    "test": "jest",
+    "test": "jest --no-colors",
```

Comment:
Not a fan of colors? I kind of like them so am kind of bummed to see this.

Reply:
It's just for the tests.  The CI server doesn't support them, whereas locally our terminals do.  It was causing test failures.